traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java

```diff
@@ -86,20 +86,16 @@ protected void main() {
         taskManager.addTask(authTokenFetchBlockTask);
     }
 
-    private static Traveler getInstance() {
-        if (localInstance == null) {
-            Log.e(TAG, "SDK Not initialized");
+    private static boolean isUninitialized() {
```

Comment:
Not trying to be a stickler but `isUninitialized` is a little weird to comprehend....`isInitialized` is much easier...

then the check becomes `!isInitialized()`

Reply:
I think when i wrote that I figured every time the check happens we're inverting, so wrote it that way. 

good suggestion though, will do the non-inverted version